dependencies/centos-7/run_time.txt

```diff
@@ -16,8 +16,11 @@ perl-Switch
 python36u
 python36u-pip
 python36u-tkinter
+qt5-qtbase
+qt5-qtimageformats
```

Comment:
I think this needs to be in compile_time.txt too.

Reply:
Anything in run_time is also in compile_time.